Added documentation for `vault operator usage`. (#10432)
hashicorp/vault: website/pages/docs/commands/operator/usage.mdx

---
layout: docs
page_title: operator usage - Command
sidebar_title: <code>usage</code>
description: |-
  "operator usage" produces a report on active clients.
---

# operator usage

The `operator usage` command allows an administrator to retrieve a
[client count](https://www.vaultproject.io/docs/concepts/client-count) report
for the default reporting period, or for a specific time range of months.

The command output will list distinct entities, non-entity tokens, and total
active clients for each namespace, and the total for the entire cluster.  It may
indicate that no data is available for the requested time range, which may be

> because the collection of client count is disabled, or the time range is too far

Reviewer: ```suggestion
because the client count reporting is disabled, the time range is too far
```

Author: Sure. I was trying to make a distinction between collection (which might be disabled) and reporting (which never is.)

Reviewer: Oh, that makes sense. Maybe `client count <some word instead of reporting>` then?
